Added "players" key display type

WeatherCats opened the pull request.

WeatherCats committed:
Added "players" key display type

ptown16 commented on src/main/java/org/cubeville/cvstats/leaderboards/Leaderboard.java:
```diff
@@ -250,6 +250,14 @@ private List<String> getLeaderboardLines() {
                 String value = leaderboardResults.getString("value");
                 if (this.key.equals("player")) {
                     key = Bukkit.getOfflinePlayer(UUID.fromString(key)).getName();
+                } else if (this.key.equals("players")) {
+                    List<String> playerNames = new ArrayList<>();
+                    for (String player : key.split(",")) {
+                        playerNames.add(Bukkit.getOfflinePlayer(UUID.fromString(player)).getName());
+                    }
+                    StringJoiner joiner = new StringJoiner(", ");
+                    playerNames.forEach(joiner::add);
+                    key = joiner.toString();
```
Could you add a sorting mechanism so that the players are ordered in the same way every time? (Maybe by sorting their UUIDs?) Otherwise, I anticipate a situation where we have a bunch of duplicate inputs for a set of players otherwise, and it'd be nice to get data based on a certain group.

WeatherCats replied:
> Could you add a sorting mechanism so that the players are ordered in the same way every time? (Maybe by sorting their UUIDs?) Otherwise, I anticipate a situation where we have a bunch of duplicate inputs for a set of players otherwise, and it'd be nice to get data based on a certain group.

I was doing that on the plugin-side implementing into it, but I suppose it would make sense to do it on this side either instead or in addition to the plugin implementing into it.

I hadn't considered that player appears in other places, my bad. I don't know why I didn't think to check. I'll work on this.

ptown16 replied:
No problem! That's what PRs are for ^^

ptown16 reviewed (commented):
May need to add something similar in a couple other places:
https://github.com/ptown16/CVStats/blob/main/src/main/java/org/cubeville/cvstats/commands/ClearMetric.java#L29
https://github.com/ptown16/CVStats/blob/main/src/main/java/org/cubeville/cvstats/commands/SendMetric.java#L26

WeatherCats committed:
Players command functionality

Added players key working with clear and send metric, also fixed bug with player key in send metric.

WeatherCats commented:
Should now work with the commands you sent me. While the commands do sort, I didn't end up adding sorting on the leaderboard's side - my logic there is that sorting the names on the display leaderboard would still just end up getting the same problem of duplicate entries, though... as I'm thinking about it, I suppose you could then go through all the entries post-sorting that would appear on the leaderboard, and get rid of duplicates? But then you'd still need more entries to display so that's not simple either. But anyway, it literally only takes like a couple lines max in the plugin using CVStats to just sort the data before it even enters the system, so I don't think it's too big of a deal currently.
